charmhelpers/contrib/network/ovs/__init__.py

```diff
@@ -134,6 +134,16 @@ def set_manager(manager):
                            'ssl:{}'.format(manager)])
 
 
+def set_value(table, arg1, arg2):
+    ''' Set the controller for the local openvswitch '''
+    log('Setting table {} {} {}'.format(table, arg1, arg2))
+    subprocess.check_call(['ovs-vsctl', 'set', table, arg1, arg2])
+
+
+def set_value_Open_vSwitch(arg):
```

Comment:
This also needs a docstring; however, if this is all that `set_value` above is being used for, then I'd ditch `set_value(...)` and merge the functionality into this one and just have a specific function.